raster/r.univar/r.univar_main.c

```diff
@@ -202,7 +213,24 @@ int main(int argc, char *argv[])
 	    }
 	}
 
+    sscanf(param.threads->answer, "%d", &threads);
+    if (threads < 1)
+    {
+      G_warning(_("<%d> is not valid number of threads. Number of threads will be set to <%d>"),
+      threads, abs(threads));
+      threads = abs(threads);
+    }
+#if defined(_OPENMP)
+    if (!param.extended->answer) {
```

Comment:
If I understand it right, reason for not using parallel version when extended is set, is lack of parallelization in calculation of extended statistics. If it is so, just add a comment here.
```suggestion
    /* Calculation of extended statistics is not parallelized yet */
    if (!param.extended->answer) {
```